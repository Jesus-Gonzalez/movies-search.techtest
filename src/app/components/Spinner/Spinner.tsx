import React from 'react';

import styles from './Spinner.module.scss';

export function RawSpinner() {
  return (
    <span className={styles.ldsDualRing} />
  )
}

export default React.memo(RawSpinner);
