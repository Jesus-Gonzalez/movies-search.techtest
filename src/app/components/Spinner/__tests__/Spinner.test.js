import React from 'react';
import renderer from 'react-test-renderer';

import { RawSpinner as Spinner } from '../Spinner';

it('render routes', () => {
  expect(renderer.create(<Spinner />))
    .toMatchSnapshot();
});
