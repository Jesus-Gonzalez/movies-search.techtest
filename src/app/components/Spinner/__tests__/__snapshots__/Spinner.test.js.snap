// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render routes 1`] = `
<span
  className="ldsDualRing"
/>
`;
